nifi-nar-bundles/nifi-influxdb-bundle/nifi-influxdb-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,28 @@
+nifi-influxdb-nar
+Copyright 2017 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+===========================================
+MIT License
+===========================================
+
+The following binary components are provided under the MIT License
+
+  The MIT License (MIT)
```

Comment:
same comment as before.  No MIT license section belongs in NOTICE.  Only LICENSE.  thanks